modules/apps/opensocial/opensocial-portlet/docroot/gadget/js/main.js

```diff
@@ -23,576 +23,624 @@ AUI.add(
 
 		var STR_EMPTY = '';
 
-		var TPL_IFRAME = '<iframe class="' + CSS_CLASS_GADGET + '" frameborder="no" {height} id="{iframeId}" name="{iframeId}" scrolling="{scrolling}" src="{src}" {width}></iframe>';
-
-		var Gadget = A.Component.create(
-			{
-				ATTRS: {
-					additionalParams: {},
-					appId: {},
-					baseRenderURL: {
-						validator: Lang.isString
-					},
-					checksum: {},
-					content: {},
-					country: {
-						setter: function(v) {
-							if (!v) {
-								v = 'ALL';
-							}
-
-							return v;
+		var TPL_IFRAME =
```

Comment:
This module is:

```
Liferay-Releng-Bundle: false
Liferay-Releng-Deprecated: true
Liferay-Releng-Marketplace: false
```

I wouldn't even bother. I'd say we can just drop this commit and let it be.

Reply:
I wonder how many of these deprecated modules I've worked on in the past couple months without even knowing they were deprecated.

Gonna have to make it a habit of checking the `app.bnd` file before working on any of the less obvious ones.

Reply:
Do we just forget this entire PR ever happened?

Reply:
No, just drop the commits for the `opensocial` and `microblogs` app which are the ones that don't really need it. The `sharing` app is still in development, so it's worth pushing it through.

Reply:
Or resend, as you see fit :)

Reply:
I'll remove the code related to `opensocial` and `microblogs` and force push the remaining code to this PR